crates/datafusion_ext/src/vars/constants.rs

```diff
@@ -233,6 +237,36 @@ pub(super) const IS_SERVER_INSTANCE: ServerVar<bool> = ServerVar {
 "#,
 };
 
+pub(super) const EXTENSION_DIRECTORY: ServerVar<str> = ServerVar {
+    name: "extension_directory",
+    value: get_extension_path(),
+    group: "glaredb",
+    user_configurable: true,
+    description: "Directory to load extensions from",
+};
+
+#[cfg(not(target_os = "windows"))]
+const fn get_home_dir() -> &'static str {
+    env!("HOME")
+}
+
+#[cfg(target_os = "windows")]
+const fn get_home_dir() -> &'static str {
+    env!("USERPROFILE")
+}
+
+// ~/.glaredb/extensions/0.1.0/macos_x86_64
+const fn get_extension_path() -> &'static str {
+    concatcp!(
+        get_home_dir(),
```

Comment:
does this mean that we're setting the extension path to the home directory on the build machine? (`/home/ubuntu` or similar?)

I would want to use whatever the default is set on the CLI parser/config file (if we had one) parser.